src/data/taskList.ts

```diff
@@ -42,3 +42,24 @@ export const businessCaseTag = (intakeStatus: string) => {
       return '';
   }
 };
+
+export const finalBusinessCaseTag = (intakeStatus: string) => {
```

Comment:
These statuses probably aren't 100% correct. We'll probably need to adjust something as we learn more about what tag will display depending on the status.